src/Module/Hooks/AdminPanelRenderService.php

```diff
@@ -135,6 +135,8 @@ public function getShipmentOptionsContext(
             }
 
             $context = array_merge($context, $this->getDeliveryOptionsContext($order, $presetDeliveryOptions));
+            $context['deliveryOptions']['carrierName'] = $context['deliveryOptions']['carrier'];
+            $context['deliveryOptions']['carrier']     = $order->getIdCarrier();
```

Comment:
zeker als je ook name gebruikt moet dit id heten.

```suggestion
            $context['deliveryOptions']['carrierId']     = $order->getIdCarrier();
```